.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,16 @@
+name: Playwright Setup
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies for all browsers
```

Comment:
@lipmaneth , very low priority, but there seems to be inconsistency in the playwright docs. It is not clear how the latest version behaves. When you add an actual test and get it to run, try to run the CI with and without this section in the action script. 

```
   run: npx playwright install-deps
```

I am not convinced that it's necessary based on the latest docs. These dependency installs can take a few minutes to run, which is why I like to keep them to a minimum.

Reply:
Ok can take a look.